src/Components/PresentationalComponents/CVEPageDetails/CVEPageDetails.js

```diff
@@ -36,7 +37,7 @@ const CVEPageDetails = props => {
                                         </GridItem>
                                         <GridItem span={1}>{}</GridItem>
                                         <GridItem span={11}>
-                                            <Title size="md">CVSS Base Score: {props.data.data.cvss3_score}</Title>
+                                            <Title size="md">CVSS Base Score: {parseCvssScore(props.data.data.cvss2_score, props.data.data.cvss3_score)}</Title>
```

Comment:
In Travis CI I see "Line 40 exceeds the maximum line length of 130  max-len" hence it does not build.